app/assets/javascripts/browsered/web-payments/format-card-types.js

```diff
@@ -9,6 +9,7 @@ module.exports = (allowedCardTypes, provider) => {
     if (type.debit || type.credit) {
       return type.brand
     }
+    return true
```

Comment:
This is going to return `true` for available networks when the type is not debit/credit. So the result could be something like `['visa', 'unionpay', true]`. We probably need to update this block to avoid returning 'true' or undefined